### PR TITLE
billing: implement billing services

### PR DESCRIPTION
**Pull request by cpanato**
This PR adds the new services to the go-github client: Billing Services

https://docs.github.com/en/rest/reference/billing

Fixes: https://github.com/google/go-github/issues/1788

/cc @gmlewis

### CODE REVIEW

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:47]:
```suggestion
// GetActionsBillingOrg returns the summary of the free and paid GitHub Actions minutes used for an Org.
```

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:61]:
```suggestion
// GetPackagesBillingOrg returns the free and paid storage used for GitHub Packages in gigabytes for an Org.
```

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:75]:
```suggestion
// GetStorageBillingOrg returns the estimated paid and estimated total storage used for GitHub Actions
// and GitHub Packages in gigabytes for an Org.
```

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:103]:
```suggestion
// GetPackagesBillingUser returns the free and paid storage used for GitHub Packages in gigabytes for a user.
```

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:117]:
```suggestion
// GetStorageBillingUser returns the estimated paid and estimated total storage used for GitHub Actions
// and GitHub Packages in gigabytes for a user.
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @cpanato ... this looks great!
Just a few minor comment tweaks, please, and then we should be ready for merging after a second LGTM.

File: github/billing.go
Comment (cpanato) [reply github/billing.go:117]: thanks for your review and feedback @gmlewis 
PTAL

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:75]: I don't think this is what I requested, @cpanato - but it is really hard to tell because you did a force push.
In the future, please don't use force pushes in PRs for this repo, as it makes it really difficult for the reviewer to see what changed.

File: github/billing.go
Comment (gmlewis) [inline github/billing.go:117]: Same here.

File: github/billing.go
Comment (gmlewis) [reply github/billing.go:75]: This is item # 7 in: https://github.com/google/go-github/blob/master/CONTRIBUTING.md#submitting-a-patch

Comment (cpanato) [reply github/billing.go:75]: sorry

Comment (cpanato) [reply github/billing.go:75]: but looking the your comments, I dont see what is wrong

File: github/billing.go
Comment (cpanato) [reply github/billing.go:117]: your suggestion was https://github.com/google/go-github/pull/1797#discussion_r571637219

File: github/billing.go
Comment (gmlewis) [reply github/billing.go]: OK, no problem.  Here is the original comment: https://github.com/google/go-github/pull/1797#discussion_r571635948

which has two lines:

```go
// GetStorageBillingOrg returns the estimated paid and estimated total storage used for GitHub Actions
// and GitHub Packages in gigabytes for an Org.
```

File: github/billing.go
Comment (gmlewis) [reply github/billing.go]: Yes, that link is correct.  It has two lines in it:

```
// GetStorageBillingUser returns the estimated paid and estimated total storage used for GitHub Actions
// and GitHub Packages in gigabytes for a user.
```

Comment (cpanato) [reply github/billing.go:118]: I think now it is ok @gmlewis and again sorry for the force push

**Review by gmlewis — APPROVED** [review]
Thank you, @cpanato !
LGTM.

Awaiting second LGTM before merging.

**Review by wesleimp — APPROVED** [review]
LGTM 👌🏼

**Comment by gmlewis** [comment]
Thank you, @wesleimp !
Merging.

**Pull request merged**
